avocado/utils/software_manager.py

```diff
@@ -9,6 +9,7 @@
 #
 # See LICENSE for more details.
 #
+# Copyright: IBM 2008-2009
 # Copyright: Red Hat Inc. 2013-2014
```

Comment:
I guess this should be 2009-2014 to preserve the previous copyright info?

Reply:
> I guess this should be 2009-2014 to preserve the previous copyright info?

Good catch!

Reply:
Going to fix it on v3.